kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -594,6 +594,14 @@ abstract class TFrontendService(name: String)
     resp
   }
 
+  override def UploadData(req: TUploadDataReq): TUploadDataResp = {
+    throw new UnsupportedOperationException("Method UploadData has not been implemented.")
+  }
+
+  override def DownloadData(req: TDownloadDataReq): TDownloadDataResp = {
+    throw new UnsupportedOperationException("Method DownloadData has not been implemented.")
```

Comment:
Use KyuubiSQLException.featureNotSupported?

Reply:
OK. But I see UnsupportedOperationException is used by many places. Should I replace them all or just for here?

Reply:
On second thought, we can use TDownloadDataResp with the error status directly, just like CancelDelegationToken

Reply:
@zhaohehuhu let's narrow the scope to the newly introduced API. "error status" idea SGTM.

Reply:
> On second thought, we can use TDownloadDataResp with the error status directly, just like CancelDelegationToken

this approach is not applicable, because the resp has no `status` field, thus we can not mark the response as failed